capstone-frontend/src/components/BookListAddDropdown.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { DropdownButton, Dropdown } from 'react-bootstrap';
+import CreateList from './CreateList'
+import '../styles/Book.css';
+
+const addBookToBookList = async (bookId, bookListJson) => {
+
+  const bookListUpdateJson = {
+    "id": bookListJson.id,
+    "add_gbookIDs": bookId,
+  }
+
+  // Update BookList in Firebase
+  fetch("/api/booklist", {
+    method: "PUT",
+    body: JSON.stringify(bookListUpdateJson)
+  });
```

Comment:
Will this have an effect on the UI? You might also want to add some error-handling logic in case the update fails.